deepchecks/tabular/checks/model_evaluation/train_test_prediction_drift.py

```diff
@@ -135,8 +135,8 @@ def __init__(
         self.aggregation_method = aggregation_method
         self.n_samples = n_samples
         self.random_state = random_state
-        if self.aggregation_method not in ('weighted', 'mean', 'none', 'max'):
-            raise DeepchecksValueError('aggregation_method must be one of "weighted", "mean", "none", "max"')
+        if self.aggregation_method not in ('weighted', 'mean', 'none', None, 'max'):
+            raise DeepchecksValueError('aggregation_method must be one of "weighted", "mean", "max", None')
```

Comment:
Doesn't this happen in other places as well?
For instance, ``feature_reduce`` in ``reduce_class.py``?

Reply:
it was changed there already